sw/device/lib/testing/test_framework/ottf_main.h

```diff
@@ -114,7 +115,11 @@ char *ottf_task_get_self_name(void);
     LOG_INFO("Starting test " #test_function_ "...");                    \
     uint64_t t_start_ = ibex_mcycle_read();                              \
     status_t local_status = INTO_STATUS(test_function_());               \
-    uint32_t cycles_ = ibex_mcycle_read() - t_start_;                    \
+    uint64_t t_end_ = ibex_mcycle_read();                                \
+    CHECK(t_end_ <= UINT32_MAX, "");                                     \
+    CHECK(t_start_ <= t_end_, "");                                       \
+    uint32_t cycles_ = (uint32_t)t_end_ - (uint32_t)t_start_;            \
+    CHECK(kClockFreqCpuHz <= UINT32_MAX, "");                            \
```

Comment:
I think this check can be removed, as an attempt to do a 64-bit division without using `udiv64_slow_into_u32` would trigger an error at linking time.

Reply:
Hmm, I'm not understanding this suggestion. The division will always be 64-bit because `kClockFreqCpuHz` is declared as `uint64_t`, right?

Reply:
It's not a 64-bit division, due to the casting, and the fact that you can link this code.
Since this [PR](https://github.com/lowRISC/opentitan/pull/11451) we can not link 64-bit division using the `/` operator. 
But feel free to leave the check anyway.